### ISSUE TITLE
[Feature req, not an issue, cant make pull request] Profiles Per Member

### ISSUE THREAD

**Issue by MonsterEnergyGang** [xSke/PluralKit#321]
In the suggestions channel in the discord server, we had posted this, but we want to put this here.

:

one of our headmates, misa, is genderfluid, and sometimes things are hard because we only set up one one proxy so it doesnt mess up our member count too much, but sometimes he wants to have the display name that goes by he/him or they/them or any pronouns along with an avatar switch, but the only kind of way to do that right now is to make a new proxy or change it manually. could there be a member command where they can have 'multiple' profiles in a sense, where you could do something like 
`pk;m misa profile sw masc`
to switch misa's display name to He/Him, and a masc looking avatar, and something similar for other things? like, you can create something like
`pk;m misa profile new masc`
and set things on it, like
`pk;m misa profile masc dn displaynameforpfhere`
and setting the avatar the same way

(reworded a bit)

- pacifica

**Comment by Qyriad** [comment]
Perhaps functionality like this could be generalized as support for subsystems?

**Comment by MonsterEnergyGang** [comment]
> Perhaps functionality like this could be generalized as support for subsystems?

yeah it could also be done that way as well, that was just our way of explaining it good

- raccoon and 007
